magento/inventory
# MSI-2161: UI Components in Store Pickup admin UI.

### Description (*)


### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2161: UI Components in Store Pickup admin UI

### Manual testing scenarios (*)

1. Create order with 'store pickup' delivery method.
2. Verify "Notify Order is Ready for Pickup" button is present on admin order page.

### Questions or comments


### Contribution checklist (*)
